EasyPost.Tests/ErrorTest.cs

```diff
@@ -31,5 +32,21 @@ public async Task TestError()
                 Assert.IsTrue(error.Errors.Count == 2);
             }
         }
+
+        [TestMethod]
+        public async Task TestNoApiKey()
+        {
+            _vcr = new TestUtils.VCR("no_api_key");
+            _vcr.SetUpTest("empty", "");
+
```

Comment:
Hmmm... we shouldn't need to invoke VCR here at all. We should really be making a new client with an empty API key.

Reply:
addressed